generated_code/srctiny_data_workspace.cpp

```diff
@@ -0,0 +1,72 @@
+	(tinytype)0.0000000000000000,	// state primal residual
+	(tinytype)0.0000000000000000,	// input primal residual
+	(tinytype)0.0000000000000000,	// state dual residual
+	(tinytype)0.0000000000000000,	// input dual residual
+	0,	// solve status
+	0,	// solve iteration
+	(tiny_VectorNx() << (tinytype)1.1000000238418579,(tinytype)1.1000000238418579).finished(),
+	(tiny_VectorNx() << (tinytype)1.1000000238418579,(tinytype)1.1000000238418579).finished(),
+	(tiny_VectorNu() << (tinytype)2.0999999046325684,(tinytype)2.0999999046325684).finished(),
+	(tiny_MatrixNxNx() << (tinytype)1.0000000000000000,(tinytype)1.0000000000000000,(tinytype)5.0000000000000000,(tinytype)2.0000000000000000).finished(),
+	(tiny_MatrixNxNu() << (tinytype)3.0000000000000000,(tinytype)4.0000000000000000,(tinytype)3.0000000000000000,(tinytype)1.0000000000000000).finished(),
+	(tiny_MatrixNuNhm1() << (tinytype)-0.5000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000).finished(),
+	(tiny_MatrixNuNhm1() << (tinytype)0.5000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000).finished(),
+	(tiny_MatrixNxNh() << (tinytype)-1.1000000238418579,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000).finished(),
+	(tiny_MatrixNxNh() << (tinytype)1.1000000238418579,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000).finished(),
+	(tiny_MatrixNxNh() << (tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000).finished(),
+	(tiny_MatrixNuNhm1() << (tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000,(tinytype)0.0000000000000000).finished(),
+	(tiny_VectorNu() << (tinytype)0.0000000000000000,(tinytype)0.0000000000000000).finished()
+};
```

Comment:
It would be very useful if we could print comments on what each member variable is.